test/library/packages/csv/CSV.chpl

```diff
@@ -0,0 +1,167 @@
+module CSV {
+  record CSVIO {
+    var ch: channel;
+    var sep: string = ",";
+    var hasHeader = true;
```

Comment:
As discussed elsewhere, we may want this to be `false` by default.

Reply:
Separately, we may want to drop the default values here, since they are defined again in the initializer arguments (or vice versa). Managing default values in multiple locations won't scale well as we add more args/fields.